python-doctest/greet.py

```diff
@@ -0,0 +1,36 @@
+# def greet(name="World"):
```

Comment:
Again, some comments would be nice to indicate what the commented out versions mean.